test-data/unit/python2eval.test

```diff
@@ -79,14 +79,18 @@ x = 1.5
 [case testAnyStr_python2]
 from typing import AnyStr
 def f(x): # type: (AnyStr) -> AnyStr
-    if isinstance(x, str):
+    if isinstance(x, bytes):
```

Comment:
This is necessary because AnyStr now has three possible types (bytes, str, unicode). Note that I must return 'foo' for bytes and str, because at runtime they are still indistinguishable! (And these tests actually run the Python code!)